README.md

```diff
@@ -7,7 +7,6 @@
 <br>
 <br>
 
-[![DeepSource](https://deepsource.io/gh/deepsourcelabs/hermes.svg/?label=active+issues&show_trend=true&token=VE2zr-CLligbX0UNUPqR9lne)](https://deepsource.io/gh/deepsourcelabs/hermes/?ref=repository-badge) [![DeepSource](https://deepsource.io/gh/deepsourcelabs/hermes.svg/?label=resolved+issues&show_trend=true&token=VE2zr-CLligbX0UNUPqR9lne)](https://deepsource.io/gh/deepsourcelabs/hermes/?ref=repository-badge)
```

Comment:
Any reason for removing the badges?

Reply:
These badges are not working. We need to activate the repo in labs.app.deepsource.com and then add that badge here. This can be done later.